### PR TITLE
Update README; fix bugs flagged by Mark

### PR DESCRIPTION
**Pull request by DiPierro**
East Palo Alto no longer uses CivicPlus. Switch the example to Menlo Park

### CODE REVIEW

File: civic_scraper/scrapers/civicplus.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
 import datetime
 import bs4
 import requests
+import pdb
```
Comment (zstumgoren) [inline civic_scraper/scrapers/civicplus.py:38]: Hey @DiPierro can you remove pdb from the commit?

File: civic_scraper/scrapers/civicplus.py [hunk 2]
```diff
@@ -272,7 +271,7 @@ def _get_all_assets(self, post_params):
                 meeting_name = committee[1]
                 payload = {'year': year, 'catID': cat_id, 'term': '', 'prevVersionScreen': 'false'}
                 response = requests.post(page, params=payload)
-                # import pdb;pdb.set_trace()
+                pdb.set_trace()
```
Comment (zstumgoren) [inline civic_scraper/scrapers/civicplus.py:274]: Also remove this set_trace() statement

**Comment by DiPierro** [comment]
Hi Serdar - noticed some bugs in this and am still trying to get 'em all straight. Will fix the two set_trace/pdb lines you flagged above.

**Comment by DiPierro** [comment]
Howdy, @zstumgoren, this PR is ready for your eyes now.

**Comment by zstumgoren** [comment]
Hey @DiPierro I just created a new `bugfixes` branch. Do you mind updating your pull request to send there as preliminary step? That way I can more easily test a few things on my end (e.g. I'd like to add test coverage on `master` that guards against regressions elsewhere after applying these updates). Thanks!

**Comment by zstumgoren** [comment]
Closing this in favor of the `refactor` branch which now addresses these issues and gets us to an official v0.1.0 release.

**Pull request closed without merging**